src/css/imports/controlbar.less

```diff
@@ -35,7 +35,7 @@
     top: -1px;
     bottom: 0;
     width: 100%;
-    height: auto;
+    height: @controlbar-height;
```

Comment:
It was set to auto because it lays out fine in other browsers, but in IE some other parts like ads use `display: table;` which will make it lay out correctly in there.  It seems like a better solution would be to have an `.jw-ie` defined in control-bar that does this work across ads and live as they'll both end up using the same set of code.